cmd/lncli/commands.go

```diff
@@ -2306,6 +2306,10 @@ var describeGraphCommand = cli.Command{
 			Name:  "render",
 			Usage: "If set, then an image of graph will be generated and displayed. The generated image is stored within the current directory with a file name of 'graph.svg'",
 		},
+		cli.BoolFlag{
+			Name:  "private",
```

Comment:
Awesome, look like you were able to squash 👍 

Ideally, you would make one commit for all the changes in `lnrpc/`, one for changes in `cmd/` and one commit for the changes to `rpcserver.go`. Not a blocker though!